Lombiq.HelpfulLibraries.Cli/License.md

```diff
@@ -0,0 +1,15 @@
+Copyright © 2011, [Lombiq Technologies Ltd.](https://lombiq.com)
```

Comment:
```suggestion
Copyright © 2022, [Lombiq Technologies Ltd.](https://lombiq.com)
```

Reply:
[The same](https://github.com/Lombiq/Helpful-Libraries/pull/136#discussion_r880389684) applies here too.